database/seeders/ApiEndpointsSeeder.php

```diff
@@ -8819,6 +8819,20 @@ private function seedSummitEndpoints()
                     IGroup::Administrators
                 ]
             ],
+            [
+                'name' => 'retrieve-qr-enc-key',
+                'route' => '/api/v1/summits/{id}/qr-codes-enc-key',
+                'http_method' => 'GET',
+                'scopes' => [
+                    sprintf(SummitScopes::ReadSummitsEncKey, $current_realm)
+                ],
+                'authz_groups' => [
```

Comment:
this is not needed we are using only service account
and not authz groups ( users )